README.md

```diff
@@ -1,79 +1,16 @@
 # template-cloud-object-storage
+# WIP DO NOT USE IN PRODUCTION
 [![Build Status](https://travis-ci.org/ibm-functions/template-cloud-object-storage.svg?branch=master)](https://travis-ci.org/ibm-functions/template-cloud-object-storage)
 
-# Work in progress !!
-
-Do not use for production.
-
 ### Overview
-This repository allows you to deploy a Cloud Object Storage Package for IBM Functions.
-The package contains a set of simple functions to get your started composing IBM Functions Applications.
-
-### Available Languages
-This template is available in Node.js 8, Node.js 6, and Python 3.
-
-# Deploy Cloud Object Storage Package with IBM Cloud Command Line Interface (CLI)
-
-## Configure CLI
-- Make sure to execute `bx login` if not already logged in
-- Install IBM Functions CLI plugin
-```
-bx plugin install cloud-functions
-```
-Make sure you are authenticated with IBM Functions and can list entities without errors
-```
-bx wsk list
-```
-## Deploy
-
-Use wskdeploy to deploy using [`manifest.yml`](./manifest.yml)
-```
-pushd runtimes/nodejs/
-PACKAGE_NAME=cloud-object-storage wskdeploy
-popd
-```
+You can use this template to deploy some IBM Cloud Functions assets for you.  The assets created by this template are described in the manifest.yaml file, which can be found at `template-cloud-object-storage/runtimes/nodejs/manifest.yaml`
```

Comment:
You can use this template to deploy IBM Cloud Functions assets for you. The assets created by this template are described in the `manifest.yaml` file, which can be found at `template-cloud-object-storage/runtimes/nodejs/manifest.yaml`.